genapi/genapi.go

```diff
@@ -712,23 +742,28 @@ func (g *GenAPI) initRedis() {
 
 func (g *GenAPI) initOkq() {
 	okqAddr, _ := g.ParamStr("--okq-addr")
+	if okqAddr == "" && g.OkqInfo.Optional {
+		return
+	}
 	okqPoolSize, _ := g.ParamInt("--okq-pool-size")
 	kv := llog.KV{
 		"addr":     okqAddr,
 		"poolSize": okqPoolSize,
 	}
 
+	if g.OkqInfo.Timeout.Nanoseconds() == 0 {
```

Comment:
`time.Duration` is a type wrapper around `int64`, so you can just do `g.OkqInfo.Timeout == 0` here.